src/useMultiRootEditor.tsx

```diff
@@ -241,7 +244,7 @@ const useMultiRootEditor = ( props: MultiRootHookProps ): MultiRootHookReturns =
 	/**
 	 * Callback function for handling a detached root.
 	 */
-	const onDetachRoot = useRefSafeCallback( ( editor: MultiRootEditor, evt: EventInfo, root: RootElement ): void => {
+	const onDetachRoot = useRefSafeCallback( ( _: MultiRootEditor, __: EventInfo, root: RootElement ): void => {
```

Comment:
Why was this change needed?

Reply:
This is a TypeScript thing. It shows warnings when there are unused function parameters and the way to remove these warnings is to rename the parameters to start with `_`. Common convention is to use `_` when you don't care about given parameter, but for readability I renamed them to `_editor` and `_evt`.